pyproject.toml

```diff
@@ -109,6 +109,9 @@ models = ["cloudpickle"]
 perf = ["orjson"]
 importers = ["polars<=1.2.1", "rich", "filelock", "mlflow", "tenacity"]
 workspaces = ["wandb-workspaces"]
+automations = [
+    "pydantic>=2.6,<3",
+]
```

Comment:
probably don't need this one anymore, as we have it by default

Reply:
Good catch thanks! (will delete soon)